scheduler/graphql_mid/types.py

```diff
@@ -105,6 +106,13 @@ def for_site(self, site: Site) -> 'SPlans':
             plans_per_site=[plans for plans in self.plans_per_site if plans.site == site])
 
 
+@strawberry.enum
```

Comment:
Just to affirm: we'll have to add event types here as we create more, right?
Could you document this process somewhere in a short blurb since we will be adding a lot more event types and I don't want to miss an essential detail?